packages/venia-styleguide/src/components/DrawerFooter/DrawerFooter.css

```
@@ -0,0 +1,9 @@
+.root {
+    align-items: center;
+    background-color: rgb(var(--venia-global-color-gray-75));
```

```suggestion
    background-color: rgb(var(--venia-global-color-gray-50));
```
---
I believe that in all cases, the drawer footer has a white background.

---

I remember picking that from another css file from the style guide. Will make the change.